server/controllers/StageController.js

```diff
@@ -46,7 +46,6 @@ class StageController {
       const trip = await this.tripController.getTrip(userId, tripId)
       const stagesPerTripId = await StageModel.find({
         tripId: new mongoose.Types.ObjectId(tripId),
-        userId: new mongoose.Types.ObjectId(userId),
```

Comment:
This will effectively undo a good chunk of my auth PR. Should only bypass userId search if u pass in a flag

Reply:
Hmmm or maybe it isn't needed since we get trip first. But just to be sure, how come u delete this line

Reply:
This might have been due to the merge conflict resolution, can undo